homeassistant/components/yeelight/__init__.py

```diff
@@ -112,8 +105,7 @@
     "hue",
     "sat",
     "color_mode",
-    "flowing",
```

Comment:
Could you explain why this two properties can be removed?

Reply:
There is some limit, how many properties can be queried at single query. That properties are not used in HA, or couldn't find any use of them. Without removing some I couldn't make all ambilight related properties to be fetched at one query. I can undo remove here, and remove in ambilight support PR.

Reply:
If the properties aren't used and aren't needed by python-yeelight the removal is fine.

Reply:
They are not needed for HA, or not used by python-yeelight. I've checked both.